.gitignore

```diff
@@ -99,3 +99,17 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# PyCharm generated files
```

Comment:
This is very minor, but since some of these are more about the developer tools that aren't specific to the project code it might be better to add them to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead.